source/_docs/energy/water.markdown

```diff
@@ -11,11 +11,13 @@ Although water is not strictly "energy", it is still a valuable resource to trac
 
 Home Assistant will need to know the amount of water that is being consumed in order to be able to track it. Several hardware options are available to do this.
 
-We have the following integrations available for existing products that can provide information about water usage:
+We have the following integrations available for existing products that can provide information about water usage, either hardware that you buy or hardware that your utility company provide:
```

Comment:
```suggestion
We have the following integrations available for existing products that can provide information about water usage:
```
Small tweak: Moved the section about hardware one line above, where hardware is already mentioned. What do you think?